README.adoc

```diff
@@ -32,6 +32,17 @@ $ yarn start # runs the handler locally
 
 Or, run `scripts/start.sh` to do this for you!
 
+== How to run the tests
+
+[Install Jest](https://jestjs.io/docs/en/getting-started) and run the tests from the `lamdba` directory:
+
+[source,bash]
+----
+$ yarn add --dev jest
```

Comment:
I don't think we need to explicitly install Jest as it is listed as a `devDependency` in package.json. So this would become "run `yarn test`".

Reply:
I think you're right! I've updated this to instruct to just install all dependencies first.

Reply:
@akash1810 @stephengeller  As it happened, when I ran `yarn test` on my machine, it did fail, and I _did_ have to execute `yarn add --dev jest` to get it to work.

I'm probably unusual, in that I'm not a regular javascript developer, so I didn't have jest already installed like you!